### ISSUE TITLE
Failing test "addToBody option with promise in error" on windows

### ISSUE THREAD

**Issue by StarpTech** [fastify/fastify-multipart#143]
## 🐛 Bug Report

The request is closed prematurely. The `finish` event of busboy is not called.

E.g https://github.com/fastify/fastify-multipart/pull/140/checks?check_run_id=1016276680 or https://github.com/fastify/fastify-multipart/runs/1016446675

```
    # Subtest: addToBody option with promise in error
        1..3
        ok 1 - client pump: no err
    # test count(7) != plan(null)
    # failed 1 of 7 tests
        # test count(1) != plan(3)
        # failed 1 test
```

After excluding the test on windows I run into a different one https://github.com/fastify/fastify-multipart/runs/1016451884?check_suite_focus=true (Looks similiar)

```
    # Subtest: should not allow __proto__
        1..5
        ok 1 - expect truthy value
        ok 2 - should be equal
        ok 3 - client pump: no err
    # test count(5) != plan(null)
    # failed 1 of 5 tests
        # test count(3) != plan(5)
        # failed 1 of 3 tests
```

## To Reproduce

Run the test in the CI or locally on Windows. You may have to run it several times ~30 in a row to reproduce it.

## Expected behavior

Should work as expected in the tests.

## Your Environment

- *node version*: v14.8.0, v12.18.3
- *fastify version*: >=3.2.1
- *os*: Windows

**Comment by mcollina** [comment]
Can we skip them on windows for now?

**Comment by StarpTech** [comment]
Yes, everything included in https://github.com/fastify/fastify-multipart/pull/140

**Comment by StarpTech** [comment]
Closed due to new api #140
